### PR TITLE
Renovate connext relayer for use in testnet prod

### PR DESCRIPTION
**Pull request by jakekidd**
## Description

Fixes https://github.com/connext/nxtp/issues/2254

See issue for description on tasking

## Type of change

- [x] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [x] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

### CODE REVIEW

**Review by wanglonghong — CHANGES_REQUESTED** [review]
https://github.com/connext/nxtp/blob/2254-renovate-relayer/packages/adapters/relayer/src/gelato/gelato.ts#L98

https://github.com/connext/nxtp/blob/2254-renovate-relayer/packages/adapters/relayer/src/connext/connext.ts#L113

It looked like we were missing to add the `res(undefined)` in above lines.

**Comment by wanglonghong** [comment]
> https://github.com/connext/nxtp/blob/2254-renovate-relayer/packages/adapters/relayer/src/gelato/gelato.ts#L98
> 
> https://github.com/connext/nxtp/blob/2254-renovate-relayer/packages/adapters/relayer/src/connext/connext.ts#L113
> 
> It looked like we were missing to add the `res(undefined)` in above lines.

Updated them in the next commits.

File: packages/agents/relayer/src/config.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const getEnvConfig = (
         process.env.SEQ_SERVER_PORT ||
         configJson.server?.port ||
         configFile.server?.port ||
-        8081,
+        8080,
```
Comment (carlomazzaferro) [inline packages/agents/relayer/src/config.ts:80]: good catch

File: ops/mainnet/prod/core/config.tf [hunk 2]
```diff
@@ -31,6 +31,18 @@ locals {
     { name = "WEB3_SIGNER_PRIVATE_KEY", value = var.sequencer_web3_signer_private_key },
     { name = "WEB3SIGNER_HTTP_HOST_ALLOWLIST", value = "*" }
   ]
+  relayer_env_vars = [
+    { name = "NXTP_CONFIG", value = local.local_relayer_config },
+    { name = "ENVIRONMENT", value = var.environment },
+    { name = "STAGE", value = var.stage },
+    { name = "DD_PROFILING_ENABLED", value = "true" },
+    { name = "DD_ENV", value = var.stage },
+    { name = "DD_SERVICE", value = "router-${var.environment}" }
```
Comment (carlomazzaferro) [inline ops/mainnet/prod/core/config.tf:40]: relayer-

File: ops/mainnet/prod/core/config.tf [hunk 3]
```diff
@@ -211,9 +223,33 @@ locals {
...
-      prover = "https://betteruptime.com/api/v1/heartbeat/${var.lighthouse_prover_heartbeat}"
+      prover    = "https://betteruptime.com/api/v1/heartbeat/${var.lighthouse_prover_heartbeat}"
       processor = "https://betteruptime.com/api/v1/heartbeat/${var.lighthouse_processor_heartbeat}"
     }
     hubDomain = "6648936"
   })
+
+  local_relayer_config = jsonencode({
+    redis = {
+      host = module.sequencer_cache.redis_instance_address,
```
Comment (carlomazzaferro) [inline ops/mainnet/prod/core/config.tf:234]: Same redis as sequencer, sure here? Don't know the architecture, just pointing it out

**Pull request merged**